### ISSUE TITLE
compile error against recent scarthgap

### ISSUE THREAD

**Issue by timblechmann** [radxa/meta-rockchip#26]
scarthgap did an upgrade of xserver-xorg from 21.1.13 to 21.1.14:
https://git.openembedded.org/openembedded-core/commit/?id=957ba32bc6fdffd3a796a04ba222fae6cd673f7e

however this meta-layer overwrites the location for the xserver:
https://github.com/radxa/meta-rockchip/blob/scarthgap-vendor/recipes-graphics/xorg-xserver/xserver-xorg_%25.bbappend#L7

while there is a tag for 21.1.13, there is none for  21.1.14:
https://github.com/JeffyCN/xorg-xserver/tree/21.1.13_2024_06_24 # exists
https://github.com/JeffyCN/xorg-xserver/tree/21.1.14_2024_06_24 # does not exist

**Comment by vbpieter** [comment]
Yeah you can downgrade the version as follows in `recipes-graphics/xorg-xserver/xserver-xorg_%.bbappend`
```patch
-SRC_URI:append = " git://github.com/JeffyCN/xorg-xserver;protocol=https;nobranch=1;branch=${PV}_2024_06_24;"
-SRC_URI:remove = "https://www.x.org/releases//individual/xserver/xorg-server-${PV}.tar.bz2"
+SRC_URI:append = " git://github.com/JeffyCN/xorg-xserver;protocol=https;nobranch=1;branch=21.1.13_2024_06_24;"
+SRC_URI:remove = "https://www.x.org/releases//individual/xserver/xorg-server-21.1.13.tar.bz2"
```
